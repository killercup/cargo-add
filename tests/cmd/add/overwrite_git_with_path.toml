bin.name = "cargo-add"
args = ["add", "cargo-list-test-fixture-dependency", "--path", "../dependency"]
status = "success"
fs.sandbox = true
fs.cwd = "overwrite_git_with_path.in/primary"

[env.add]
CARGO_IS_TEST="1"
