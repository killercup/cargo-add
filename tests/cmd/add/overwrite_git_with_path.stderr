      Adding cargo-list-test-fixture-dependency (unknown version) to dependencies. 
